### DISCUSSION TITLE
Unable to access Github copilot after being approved.

### DISCUSSION THREAD

**Discussion by Jossyboydgenius** [community/community#118346] [GitHub Education]
### Hi everyone,

I received an e-mail that I've been approved....
My profile switched to pro but the most important thing to me which is the copilot is not accessible.
What could be the issue, please? 
<img width="124" alt="Screenshot 2024-04-08 at 8 46 28 PM" src="https://github.com/community/community/assets/130408142/ccbf42cf-248c-40cd-8207-bd18bc6b2d86">
<img width="1280" alt="Screenshot 2024-04-08 at 10 42 21 PM" src="https://github.com/community/community/assets/130408142/e0c92156-d193-4046-bbc5-3fa8ab6c869d">
<img width="1280" alt="Screenshot 2024-04-08 at 10 42 47 PM" src="https://github.com/community/community/assets/130408142/5a785e74-4612-4e4b-8a35-f68d39d18539">

**Comment by jfullstackdev** [comment]
as far as I know, and Copilot confirmed it, it is not included in the Pro Subscription, even the website says it, it's a separate billing

![image](https://github.com/community/community/assets/47092464/99fa7008-dc49-4e44-86ad-97649707de11)
